ios/RCTTWVideoModule.m

```diff
@@ -450,17 +460,23 @@ -(NSMutableDictionary*)convertLocalVideoTrackStats:(TVILocalVideoTrackStats *)st
       builder.preferredVideoCodecs = @[ [TVIH264Codec new] ];
     }
 
-    if(encodingParameters[@"audioBitrate"] || encodingParameters[@"videoBitrate"]){
-      NSInteger audioBitrate = [encodingParameters[@"audioBitrate"] integerValue];
-      NSInteger videoBitrate = [encodingParameters[@"videoBitrate"] integerValue];
-      builder.encodingParameters = [[TVIEncodingParameters alloc] initWithAudioBitrate:(audioBitrate) ? audioBitrate : 40 videoBitrate:(videoBitrate) ? videoBitrate : 1500];
-    }
-
     if (enableNetworkQualityReporting) {
       builder.networkQualityEnabled = true;
       builder.networkQualityConfiguration = [ [TVINetworkQualityConfiguration alloc] initWithLocalVerbosity:TVINetworkQualityVerbosityMinimal remoteVerbosity:TVINetworkQualityVerbosityMinimal];
     }
 
+    if(videoParams[@"width"] && videoParams[@"height"]){
+      NSInteger width = [videoParams[@"width"] integerValue];
+      NSInteger height = [videoParams[@"height"] integerValue];
+      kRCTTWVideoAppCameraSourceDimensions.width = width;
+      kRCTTWVideoAppCameraSourceDimensions.height = height;
+      builder.encodingParameters = [[TVIEncodingParameters alloc] initWithAudioBitrate: 16 videoBitrate:(width <= 640) ? 512 : 5000];
```

Comment:
please use param VideoQuality.LOW like ionic

Reply:
in ionic we also have something for downstream bitrate, please copy paste that too.

Reply:
in capacitor plugin we also used 512: https://github.com/media4care/capacitor-twilio-video/blob/master/android/src/main/java/com/media4care/twilio/video/TwilioVideoActivity.java